client/components/carousel.jsx

```diff
@@ -43,7 +43,7 @@ class Carousel extends React.Component {
   render() {
     if (this.props.displayCarousel) {
       return (
-        <Modal>
+        <Modal tabIndex={0} onKeyDown={(event) => this.props.handleKeyPress(event)}>
           <CloseButton onClick={this.props.closeCarousel}>{this.props.closeSymbol}</CloseButton>
```

Comment:
Question: are you using destructuring for props? 

(My linter has been demanding it)

Reply:
No, I haven't been. My linter hasn't been demanding it from me...I wonder why